api/src/main/resources/messages_fr.properties

```diff
@@ -248,5 +248,8 @@ ${project.parent.artifactId}.report.programs.datasetDescription=Programs SQL Dat
 ${project.parent.artifactId}.report.appointments.reportName=Appointments Report
 ${project.parent.artifactId}.report.appointments.reportDescription=Appointments Report
 
+${project.parent.artifactId}.report.audits.reportName=Audit Report
+${project.parent.artifactId}.report.audits.reportDescription=Audit Report
```

Comment:
Shouldn't this be in french?
Seems like the other strings aren't translated as well.